internal/storer/storer.go

```diff
@@ -17,6 +17,8 @@ const (
 	DefaultTTL = 10 * time.Minute
 	// DefaultInterval is default interval to execute the "garbage collection" of the cache.
 	DefaultInterval = 15 * time.Minute
+	// NamespaceStoreExtraInterval is the extra time added to the integration interval to execute the "gc"
+	NamespaceStoreExtraInterval = 5 * time.Second
```

Comment:
Maybe it would make sense to define this in `discovery` package (in the same place the NamespaceFilter is defined).  I think defining it here feels like coupling the storer with a particular instance of the storer, WDYT?

Reply:
Yes, this is temporary, I still need to see how does the other branch evolve, since maybe there is a custom interface for the ns storer.